scripts/build_plugin_linux.sh

```diff
@@ -6,16 +6,21 @@ SCRIPTPATH=`dirname $(dirname $(realpath $0))`
 
 PLUGIN_DIR="plugin-build"
 PLUGIN_ABS_PATH=""
-
 #need to clear the LD_LIBRARY_PATH to avoid lib conflict
 unset LD_LIBRARY_PATH
+FOLDER_PKG="/trelis-sdk"
```

Comment:
This script only works in the docker build as launched in your other script, right?